NuGet/Transformations/ApplicationInsights.config.uninstall.xdt

```diff
@@ -0,0 +1 @@
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
  <TelemetryChannel xdt:Transform="Remove" xdt:Locator="Match(Type)" Type="Glimpse.ApplicationInsights.GlimpseTelemetryChannel, Glimpse.ApplicationInsights" />
</ApplicationInsights>
```

Comment:
I am removing the channel that I added in the install.xdt file. What do you mean by removing existing channel?